dist/docker/redhat/build_docker.sh

```diff
@@ -136,6 +112,12 @@ bconfig --label org.opencontainers.image.ref.name="ScyllaDB"
 bconfig --label org.opencontainers.image.version="$version-$release"
 bconfig --label description="ScyllaDB Open Source"
```

Comment:
It's not Open Source

Reply:
Probably should be `ScyllaDB source-available`

Reply:
ScyllaDB is enough

Reply:
Updated to `bconfig --label description="ScyllaDB"`